src/software/ai/hl/stp/play/play.cpp

```diff
@@ -39,6 +39,31 @@ std::vector<std::shared_ptr<Tactic>> Play::getTactics(const World &world)
     return std::vector<std::shared_ptr<Tactic>>();
 }
 
+std::vector<std::unique_ptr<Intent>> Play::get(
+    RobotToTacticAssignmentFunction robot_to_tactic_assignment_algorithm,
+    MotionConstraintBuildFunction motion_constraint_builder, const World &new_world)
+{
+    std::vector<std::unique_ptr<Intent>> intents;
+    auto tactics = getTactics(new_world);
+    std::vector<std::shared_ptr<const Tactic>> const_tactics;
+    const_tactics.reserve(tactics.size());
+    std::transform(tactics.begin(), tactics.end(), std::back_inserter(const_tactics),
+                   [](std::shared_ptr<Tactic> tactic) { return tactic; });
```

Comment:
Probably worth adding a comment here explaining why we need this `std::transform`.

Reply:
done